config/puma.rb

```diff
@@ -1,4 +1,9 @@
+require 'bunny'
+
 bind 'tcp://0.0.0.0:8080'
 workers 1
 threads 0, 10
 quiet true
+
+on_worker_boot do
+end
```

Comment:
Do nothing? Why is this necessary?

Reply:
It is not, it was a hasty commit, sadly.

This was an optional place to put the initialization of the rabbitmq client.